### PR TITLE
Reduce memory usage by removing softmax in logits2pred()

### PR DESCRIPTION
**Pull request by ivantyj**
As mentioned in #92 removing `softmax()` don't affect current logic, but will greatly reduce memory usage especially when segmenting larger CT scans using the whole-body-v2.0.1.

### CODE REVIEW

**Comment by lassoan** [comment]
@diazandr3s could you have a look at this? Thanks!
